label_studio/frontend/src/components/LanguagePicker/LanguagePicker.js

```diff
@@ -0,0 +1,30 @@
+import React, { useState } from 'react';
+import { Block } from '../../utils/bem';
+import { Button } from '../../components';
+import './LanguagePicker.styl';
+import i18n from "i18next";
+
+export const LanguagePicker = () => {
+
+  const [language, setLanguage] = useState(i18n.language);
+
+  const changeLanguage = (e) => {
+    e.preventDefault();
+    setLanguage(e.target.value);
+    i18n.changeLanguage(e.target.value);
+  };
+
+  return (
+    <Block tag="li" name="language-picker">
+      <Button size="small" type={language === 'en' ? "button" : "text"} value='en' onClick={changeLanguage}>
+        English
+      </Button>
+      <Button size="small" type={language === 'pt' ? "button" : "text"} value='pt' onClick={changeLanguage}>
+        Português
+      </Button>
+      <Button size="small" type={language === 'zh' ? "button" : "text"} value='zh' onClick={changeLanguage}>
```

Comment:
To be more accurate, Chinese Simplified should be using `zh-CN` or `zh-Hans`:

```suggestion
      <Button size="small" type={language === 'zh-CN' ? "button" : "text"} value='zh' onClick={changeLanguage}>
```

Reply:
请问您的汉化能迁移到1.21.0版本吗